pkgs/applications/networking/p2p/freenet/default.nix

```diff
@@ -1,72 +1,64 @@
-{ lib, stdenv, fetchurl, fetchFromGitHub, ant, jdk, bash, coreutils, substituteAll }:
+{ lib, stdenv, fetchurl, jdk, bash, coreutils, substituteAll }:
 
 let
+  version = "build01494";
```

Comment:
We should maybe strip the build prefix

Reply:
will do with the upcoming change to build this from source.